backend/app/auth/exceptions.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class AuthInvalidCredentialsException(ABC, BaseException):
```

Comment:
If you're going to throw exceptions I'd probably just have a single `AuthException` on the `AuthService` layer and call it a day. The specialized exceptions aren't adding information.